config/crd/v2alpha1/bases/velero.io_datauploads.yaml

```diff
@@ -115,6 +115,8 @@ spec:
               snapshotType:
                 description: SnapshotType is the type of the snapshot to be backed
                   up.
+                enum:
+                - CSI
```

Comment:
Not sure if we really want this 
cc @Lyndon-Li

Reply:
The code implicitly tell us that. If the intent is to let an open string field, would remove this type from apis file

Reply:
I believe datauploads CRD is meant for any data movers. CSI being the first one implemented by velero. Others could come in with Database snapshot uploader etc..

Reply:
I don't think we wanna introduce frequent changes to APIs.

Reply:
+1 @kaovilai

Reply:
This enum needs to be removed IMO. It's an open value per design.. or else comment in the design that future datauploaders need to add values to this enum

Reply:
Can you point to the design?

I found this, which implies it is a enum https://github.com/vmware-tanzu/velero/blob/main/design/volume-snapshot-data-movement/volume-snapshot-data-movement.md?plain=1#L162

Reply:
>  SnapshotType is the type of the snapshot to be backed up.

https://github.com/vmware-tanzu/velero/blob/e65ef2894868242d51a92e1fba26423adb6ca49f/design/volume-snapshot-data-movement/volume-snapshot-data-movement.md?plain=1#L272-L275C25

Plan is to open up this to more values in the future.